scripts/test_docker_image.sh

```
@@ -129,20 +195,32 @@
+  run_in_container "/app/bin/wanderer_notifier eval 'IO.puts \"Version test\"'" "-e DISCORD_BOT_TOKEN=$DISCORD_TOKEN -e WANDERER_ENV=test" || echo "Application eval failed, but continuing..."
+
+  echo "Testing simplified application boot..."
+  # Try to run a very simple command
+  run_in_container "/app/bin/wanderer_notifier eval 'System.version |> IO.puts'" "-e DISCORD_BOT_TOKEN=$DISCORD_TOKEN -e WANDERER_ENV=test" || echo "Simple boot test failed, but continuing..."
   
   echo "Testing minimal application boot (with clean shutdown)..."
-  # Set a lower timeout to prevent hanging if there's an issue
-  run_in_container "timeout 10 /app/bin/wanderer_notifier eval 'IO.puts(\"Application started\"); :init.stop()'" "-e DISCORD_BOT_TOKEN=$DISCORD_TOKEN -e CONFIG_PATH=/app/etc/wanderer_notifier.exs -e WANDERER_ENV=test -e WANDERER_FEATURE_DISABLE_WEBSOCKET=true"
+  # Use a shorter timeout and force kill if needed
+  run_in_container "timeout --kill-after=5s 10s /app/bin/wanderer_notifier eval 'IO.puts(\"Application started\"); Process.sleep(1000); :init.stop()'" "-e DISCORD_BOT_TOKEN=$DISCORD_TOKEN -e WANDERER_ENV=test -e WANDERER_FEATURE_DISABLE_WEBSOCKET=true" || {
+    if [ $? -eq 124 ] || [ $? -eq 137 ]; then
```

Shellcheck (0.10.0)

[warning] This $? refers to a condition, not a command. Assign to a variable to avoid it being overwritten.

(SC2319)